src/Examples/Examples.GameOfLife/Main.fs

```diff
@@ -21,16 +18,6 @@ module Main =
     | StartEvolution
     | StopEvolution
 
-    let timer (state: State) =
```

Comment:
I guess that's by mistake ?

Reply:
It is not used here, the same code is defined in `Program.fs`.

Reply:
ok